common/religion/holy_sites/wc_holy_sites.txt

```diff
@@ -0,0 +1,1326 @@
+		stewardship = 1
+	}
+}
+edj = {
+	county = c_edj
+	
+	character_modifier = {
+		name = holy_site_edj_effect_name
+		diplomacy_per_piety_level = 1
+		naval_movement_speed_mult = 0.15
+	}
+}
+#gadgetzan
+
+##################
+# shamanism_group
+##################
+### orcish_shamanism
+orgrimmar = {
+	county = c_orgrimmar
+	
+	character_modifier = {
+		name = holy_site_orgrimmar_effect_name
+		martial = 1
+		diplomacy = 1
+		controlled_province_advantage = 4
+	}
+}
+#dark_portal
+#blackrock_depths
+crossroads = {
+	county = c_crossroads
+	
+	character_modifier = {
+		name = holy_site_crossroads_effect_name
+		tax_mult = 0.2
+	}
+}
+
+### shamanism
+#grim_batol
+#ironforge
+#red_desert
+aerie_peak = {
+	county = c_aerie_peak
+	
+	character_modifier = {
+		name = holy_site_aerie_peak_effect_name
+		martial = 1
+		movement_speed = 0.05
+	}
+}
+thundermar = {
+	county = c_thundermar
+	
+	character_modifier = {
+		name = holy_site_thundermar_effect_name
+		opinion_of_same_culture = 10
+	}
+}
+
+### earth_mother_worship
+#thunder_bluff
+bloodhoof = {
+	county = c_bloodhoof
+	
+	character_modifier = {
+		name = holy_site_bloodhoof_effect_name
+		martial = 1
+		dynasty_opinion = 10
+	}
+}
+darkcloud = {
+	county = c_darkcloud
+	
+	character_modifier = {
+		name = holy_site_darkcloud_effect_name
+		intrigue = 1
+		tyranny_loss_mult = 0.1
+	}
+}
+#nordrassil
+#maraudon
+
+### tidemother
+tol_barad = {
+	county = c_tol_barad
+	
+	character_modifier = {
+		name = holy_site_tol_barad_effect_name
+		dread_gain_mult = 0.2
+		prisoner_opinion = 10
+		garrison_size = 0.2
+	}
+}
+#balor
+#boralus
+storm_isle = {
+	county = c_storm_isle
+	
+	character_modifier = {
+		name = holy_site_storm_isle_effect_name
+		dread_baseline_add = 5
+		martial_per_prestige_level = 1
+	}
+}
+#gilneas
+
+### cult_of_the_candle
+#blackrock_depths
+#ironforge
+#grim_batol
+#red_desert
+#gnomeregan
+
+### primitive_shamanism
+#stormwind
+#karazhan
+#blackrock_depths
+moonbrook = {
+	county = c_moonbrook
+	
+	character_modifier = {
+		name = holy_site_moonbrook_effect_name
+		stewardship = 2
+		farmlands_advantage = 4
+		plains_advantage = 4
+	}
+}
+#red_desert
+
+### tuskarr_religion
+njord = {
+	county = c_njord
+	
+	character_modifier = {
+		name = holy_site_njord_effect_name
+		embarkation_cost_mult = -0.25
+		stewardship_per_prestige_level = 1
+	}
+}
+moaki = {
+	county = c_moaki
+	
+	character_modifier = {
+		name = holy_site_moaki_effect_name
+		health = 0.1
+		martial_per_piety_level = 1
+	}
+}
+riplash = {
+	county = c_riplash
+	
+	character_modifier = {
+		name = holy_site_riplash_effect_name
+		monthly_prestige_gain_mult = 0.1
+	}
+}
+kamagua = {
+	county = c_kamagua
+	
+	character_modifier = {
+		name = holy_site_kamagua_effect_name
+		diplomacy_per_piety_level = 1
+		vassal_tax_mult = 0.05
+	}
+}
+#wintergarde
+
+### taunka_religion
+icemist = {
+	county = c_icemist
+	
+	character_modifier = {
+		name = holy_site_icemist_effect_name
+		army_maintenance_mult = -0.01
```

Comment:
Honestly, -1% sounds like a joke.

Reply:
Changed in afec05ea2d8b522d7578c3782f518813f1ee6ec4.